### DISCUSSION TITLE
SSL Certificate - Manual Configuration

### DISCUSSION THREAD

**Discussion by digismart0rari** [globaleaks/globaleaks-whistleblowing-software#3941] [General]
Hi, everyone
I have problem with the generation about **Certificate Signing Request (PEM)** during the manual configuration on GlobaLeaks.
I write all info correctly but it's return an generic error. Some ideas?

**Comment by evilaliv3** [comment]
Thank you. This issue should be fixed in current latest release(4.13.3).

Which version are you running?
